sys-apps/config-site/files/config.site.in

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+for script in "@datarootdir@/config.site.d"/*.conf; do
+	test -f "${script}" || continue
+	test -r "${script}" || continue
+	echo "${0##*/}: loading site script ${script}"
```

Comment:
```suggestion
	echo "${0##*/}: loading site script ${script}" >&2
```

perhaps.

Reply:
A similar message is output by configure itself (see below). The message is output to file descriptors 5 and 6. File descriptor 5 is config.log, and file descriptor 6 is a copy of stdout (fd 1). Messages are only written to stderr if an error occurs.

I think we should keep it as-is, or use the same output logic that configure uses. 

```
for ac_site_file in $ac_site_files
do
  case $ac_site_file in #(
  */*) :
     ;; #(
  *) :
    ac_site_file=./$ac_site_file ;;
esac
  if test -f "$ac_site_file" && test -r "$ac_site_file"; then
    { printf "%s\n" "$as_me:${as_lineno-$LINENO}: loading site script $ac_site_file" >&5
printf "%s\n" "$as_me: loading site script $ac_site_file" >&6;}
    sed 's/^/| /' "$ac_site_file" >&5
    . "$ac_site_file" \
      || { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
printf "%s\n" "$as_me: error: in \`$ac_pwd':" >&2;}
as_fn_error $? "failed to load site script $ac_site_file
See \`config.log' for more details" "$LINENO" 5; }
  fi
done
```

Reply:
Oh, I should also note that fd 6 is redirected to /dev/null if `--quiet` or `--silent` is passed to configure.

Do we think it is safe to treat fd 5 and 6 as a stable logging API provided by configure? If so, I will emulate the behavior seen above by writing the message to both places.

Reply:
I did a little digging in autoconf.git, and it looks like these file descriptors have been shuffled around a couple of times.

File descriptor 5 has been used for config.log since https://github.com/autotools-mirror/autoconf/commit/53b55a54e289646226661669c2d2d6f44496c195.

File descriptor 6 has been used for message output since https://github.com/autotools-mirror/autoconf/commit/f5cee524913165f3ef1a6b5700f0d485c2852cdc. Before then, file descriptor 4 was used.

Today, these are defined by the AS_MESSAGE_LOG_FD and AS_MESSAGE_FD macros, which we can't use directly from a shell script like config.site.

Reply:
> Do we think it is safe to treat fd 5 and 6 as a stable logging API provided by configure? If so, I will emulate the behavior seen above by writing the message to both places.

I dug around, but decide this isn't safe enough for me to suggest it.

We could just ask on autoconf@gnu.org, naturally, to see if there's a recommended approach.